app/src/main/java/org/sopt/and/presentation/mypageScreen/MypageViewModel.kt

```diff
@@ -0,0 +1,58 @@
+package org.sopt.and.presentation.mypageScreen
+
+import android.util.Log
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.launch
+import org.sopt.and.data.datalocal.datasource.UserInfoLocalDataSource
+import org.sopt.and.data.dataremote.network.ServicePool
+import org.sopt.and.domain.model.User
+import org.sopt.and.domain.usecase.GetUserHobbyUseCase
+import javax.inject.Inject
+
+@HiltViewModel
+class MypageViewModel @Inject constructor(
+    private val userInfoLocalDataSource: UserInfoLocalDataSource,
+    private val getUserHobbyUseCase: GetUserHobbyUseCase
+) : ViewModel() {
+
+//    private val userService by lazy { ServicePool.userService }
+
+    private val _user = MutableStateFlow(User())
+    val user: StateFlow<User> = _user
+
+    init {
+        loadUserData()
+    }
+
+    fun loadUserData() {
+        viewModelScope.launch {
+            val userName = userInfoLocalDataSource.userName
+            val accessToken = userInfoLocalDataSource.accessToken
+            val hobby = getUserHobby()
+            _user.value.name = userName
+            _user.value.hobby = hobby
+            _user.value.accessToken = accessToken
```

Comment:
```suggestion
            _user.value = _user.value.copy(
            name = userName,
            hobby = hobby,
            accessToken = accessToken
)
```
직접 필드 수정은 권장되지 않는걸로 알아서 요렇게 해보는건 어때요?